contract/areas.go

```diff
@@ -9,11 +15,23 @@ type Area struct {
 
 // GetAreasRequest defines the schema for the GET /areas query parameter
 type GetAreasRequest struct {
-	Dataset  string `schema:"dataset"`
-	AreaType string `schema:"area_type"`
+	QueryParams
 	Category string `schema:"q"`
```

Comment:
is this tag for the struct supposed to be "schema:q"? if so what does this represent? 
I would have assumed some json tag instead

Reply:
This isn't something I've changed, but the `schema` tag is for automatically decoding query params using the `gorilla/schema` package.